README.md

```diff
@@ -56,6 +73,13 @@ scrolling the area by dragging its empty space, but keep the
 opportunity to select the text (see
 [example](http://asvd.github.io/jailed/demos/web/process/)).
 
--
+Thank you for use this lib.
+
+TO DO
+==========
+
+* Edit image movement range `success`
+
+Follow me on instagram: https://www.instagram.com/cxymxa
```

Comment:
Your instagram is cool, but is not related to the programming or something like that